securedrop_client/queue.py

```diff
@@ -54,6 +76,19 @@ def _process(self, exit_loop: bool) -> None:
 
 
 class ApiJobQueue(QObject):
+    # These are the priorities for processing jobs.
+    # Lower numbers corresponds to a higher priority.
+    JOB_PRIORITIES = {
+        # LogoutJob: 1,  # Not yet implemented
```

Comment:
I think we should use a number here that starts somewhere closer to the center of the 0-50 range just to future proof this a little.